Packs/Indeni/Integrations/Indeni/Indeni.py

```diff
@@ -0,0 +1,360 @@
+from CommonServerPython import *
+''' IMPORTS '''
+
+import json
+import requests
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+''' GLOBALS/PARAMS '''
+MAX_BATCH_SIZE = 20
```

Comment:
please add this as an integration parameter that its display will be "Number of alerts to pull per API call" with 20 as the default.

Reply:
Done